integration/swarm.bats

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bats
+# *-*- Mode: sh; sh-basic-offset: 8; indent-tabs-mode: nil -*-*
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Swarm testing
+
+# Environment variables
+DOCKER_EXE="docker"
+# Image for swarm testing
+nginx_image="gabyct/nginx"
```

Comment:
why this image is pulled from your personal account?

Reply:
@MarioCarrilloA there is an issue with the nginx image of docker...this problem was reported long time ago

Reply:
oh I see, then maybe that nginx is a customized image, if yes, maybe it should live here: https://hub.docker.com/u/clearcontainers/